apps/sms/js/thread_list_ui.js

```diff
@@ -375,11 +367,6 @@ var ThreadListUI = {
         return list;
       }, []);
 
-      // That means that we've just removed some drafts
```

Comment:
anyway if I can finish my db experiment we'll get rid of the existing drafts db :) so better not think too much about this case.